websrc/source/documents/tutorials/prometheus-tutorial.md

```diff
@@ -133,6 +133,10 @@ Now if we go back to the Grafana Dashboard, we should see that the number of pac
 
 ![Basic Ping](basic_ping.png)
 
+The way the data is displayed shows a monotonically increasing counter. This may not be the most ideal way to display your data, in which case you can also graph deltas. `idelta()` is a query function that takes a range vector of the metric you want to graph. A range vector is a list of datapoints over a period of time. For example, you can graph the received bytes over a one day span with the following query.
+
+`idelta(rx_packets{containerName="contiv-c1"}[1d])`
```

Comment:
I just put in a random time frame. In general the user can put any time frame in (i.e [15m], [10d], [1w])